include/edm4hep/Constants.h

```diff
@@ -21,7 +21,7 @@
 
 namespace edm4hep {
 static constexpr const char* CellIDEncoding = "CellIDEncoding";
-static constexpr const char* EventHeader = "EventHeader";
+static constexpr const char* EventHeaderName = "EventHeader";
```

Comment:
```suggestion
static constexpr const char* eventHeader = "EventHeader";
```

As a "less-typing necessary" variant? Or maybe `EvtHeader`?

Reply:
I would prefer clarity over shortness

Reply:
I'm all for typing less 🤓 but in this case a mistake from `eventHeader` to `EventHeader` can even make the code compile and have some problems somewhere else and `EvtHeader` is also quite similar. I was going to say let Leonhard decide, but he beat me to it so it's two votes against one. I checked the CI failure and I'm quite sure it's not transient, but it's unrelated. I'll have a look and try to fix it